database/postgres/pg.js

```diff
@@ -61,4 +60,18 @@ pgModule.deletePgAccount = async (username) => {
   }
 };
 
+pgModule.userHasPgAccount = async username => {
+  if (!username) return false;
+
+  try {
+    const user = await escape(`SELECT 1 FROM `, username, ` WHERE `, username);
```

Comment:
this query doesn't look right.

Reply:
please test your code! There's no way this could work. https://stackoverflow.com/questions/8546759/how-to-check-if-a-postgres-user-exists